examples/interactive_bluefog_helloworld.ipynb

```diff
@@ -573,6 +573,98 @@
     "plt.show()"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Interrupt the hanged worker process\n",
+    "\n",
+    "It is quite common in the code dealing with the multiple machine communication that some unexpected bugs or deadlock caused the worker process(es) hanged forever. Recall the worker processes are the seperated environment from the notebook. Hence, the interrupt kernel in the tool bar of notebook can interrupt the code of execution in notebook but the worker processes are still running."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's use sleep function to simulate that situation. If you run the following code and click the interrupt button afterwards, you will be able to execute other cells but the workers still continues to sleep, which will block the running of other requests."
```

Comment:
- Use some special function for `sleep`.
-  still continues -> still continue

Reply:
Done